build-scripts/cassandra-dtest-pytest.sh

```diff
@@ -73,6 +74,8 @@ elif [ "${DTEST_TARGET}" = "dtest-offheap" ]; then
     pytest -vv --log-level="INFO" --use-vnodes --num-tokens=32 --use-off-heap-memtables --junit-xml=nosetests.xml --junit-prefix=${DTEST_TARGET} -s --cassandra-dir=$CASSANDRA_DIR --skip-resource-intensive-tests 2>&1 | tee -a ${WORKSPACE}/test_stdout.txt
 elif [ "${DTEST_TARGET}" = "dtest-large" ]; then
     pytest -vv --log-level="INFO" --use-vnodes --num-tokens=32 --junit-xml=nosetests.xml --junit-prefix=${DTEST_TARGET} -s --cassandra-dir=$CASSANDRA_DIR --only-resource-intensive-tests 2>&1 | tee -a ${WORKSPACE}/test_stdout.txt
+elif [ "${DTEST_TARGET}" = "dtest-upgrade" ]; then
+    pytest -vv --log-level="INFO" --execute-upgrade-tests --junit-xml=nosetests.xml --junit-prefix=${DTEST_TARGET} -s --cassandra-dir=$CASSANDRA_DIR 2>&1 | tee -a ${WORKSPACE}/test_stdout.txt
```

Comment:
Does this need `--use-vnodes` ?

Reply:
don't think so. The upgrade tests in CircleCI also test with vnodes disabled